jmclient/test/test_walletservice.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import (absolute_import, division,
+                        print_function, unicode_literals)
+from builtins import * # noqa: F401
+'''Tests of functionality at walletservice layer.'''
+
+import os
+
+import pytest
+from jmbase import get_log
+from jmclient import load_program_config, jm_single, \
+     WalletService
+from test_blockchaininterface import sync_test_wallet
+from test_wallet import fund_wallet_addr, get_populated_wallet
+testdir = os.path.dirname(os.path.realpath(__file__))
+log = get_log()
+
+def set_freeze_reuse_config(x):
+    jm_single().config.set('POLICY', 'min_sats_freeze_reuse', str(x))
+
+def try_address_reuse(wallet_service, idx, funding_amt, config_threshold,
+                      expected_final_balance):
+    set_freeze_reuse_config(config_threshold)
+    # check that below the threshold on the same address is not allowed:
+    fund_wallet_addr(wallet_service.wallet,wallet_service.get_addr(0, 0, idx),
```

Comment:
Add a space after comma here

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/471/commits/794726a718b902eb7e6b3d883a3f9aa92bddede0